test_runner/src/main/kotlin/ftl/config/FtlConstants.kt

```diff
@@ -69,18 +68,20 @@ object FtlConstants {
     }
 
     val defaultCredentialPath: Path by lazy {
-        Paths.get(System.getProperty("user.home"), ".config/gcloud/application_default_credentials.json")
+        if (isWindows) Paths.get(System.getenv("HOMEPATH"), ".config/gcloud/application_default_credentials.json")
+        else Paths.get(System.getProperty("user.home"), ".config/gcloud/application_default_credentials.json")
     }
 
     val credential: GoogleCredentials by lazy {
         when {
             useMock -> GoogleCredentials.create(AccessToken("mock", Date()))
             UserAuth.exists() -> UserAuth.load()
-            else -> try {
+            else -> runCatching {
                 GoogleApiLogger.silenceComputeEngine()
                 ServiceAccountCredentials.getApplicationDefault()
-            } catch (e: IOException) {
-                throw FlankGeneralError("Error: Failed to read service account credential.\n${e.message}")
+            }.getOrElse {
+                if (isWindows) GoogleCredentials.fromStream(defaultCredentialPath.toFile().inputStream())
```

Comment:
What will be the error message when flank will fail reading credentials on Windows as well?

Reply:
Probably similiar to previous one from `ServiceAccountCredentials.getApplicationDefault()`

Reply:
Thanks! I changed it to throws FlankGeneralException like on non-windows.